apps/mobile/src/components/secure-your-wallet/skip-secure-wallet-sheet.tsx

```diff
@@ -0,0 +1,22 @@
+import { RefObject } from 'react';
+
+import { t } from '@lingui/macro';
+
+import { SheetRef } from '@leather.io/ui/native';
+
+import { WarningSheet } from '../sheets/warning-sheet.layout';
+
+interface SkipSecureWalletSheetProps {
+  sheetRef: RefObject<SheetRef>;
+  onSubmit(): unknown;
```

Comment:
```suggestion
  onSubmit(): void;
```
onSubmit doesn't expect a return value